runtime/khala/src/lib.rs

```diff
@@ -1941,6 +1948,13 @@ impl_runtime_apis! {
         }
     }
 
+    impl sygma_runtime_api::SygmaBridgeApi<Block> for Runtime {
+		fn is_proposal_executed(nonce: DepositNonce, domain_id: DomainID) -> bool {
+            // TODO: enable Sygma
```

Comment:
@tolak just out of curiosity, here for phala and khala, you didn't impl the api call body, but in thala runtime you impl it, do you plan to add this when later you guys going to launch on Kusama and Polkadot? or there is some other reason?

Reply:
@freddyli7 Yes, will implement it when we enable Sygma on Kusama and Polkadot parachain